src/modules/showLikes.js

```diff
@@ -0,0 +1,36 @@
+import {
+  getLikesData,
+  sendLikesData,
+} from './likesFetch.js';
+import likesData from './likesDataStore.js';
+
+export default class TheLikes {
+  likesStoredData = JSON.parse(localStorage.getItem('likesData'))
+
+  postLike = async (pid) => {
+    let newlike = {};
+    const item = (pid).toString();
+    newlike = {
+      item_id: item,
+    };
+    await sendLikesData(newlike);
+  }
+
+    getLikes = async () => {
+      const likedValues = await getLikesData()
+        .then((res) => {
+          res.sort((task1, task2) => task1.item_id - task2.item_id);
+          likesData(res);
+          return res;
+        });
+      return likedValues;
+    };
+
+    getNewLike = async (itemId) => {
+      await this.postLike(itemId);
+      await this.getLikes();
+      // await this.likesUpdate(itemId);
```

Comment:
- [ ] Good job removing previous comments but this single line will reproduce the same effects the previewer mentioned, making your code messy. Kindly remove it pls :pray: